clouddriver-oracle-bmcs/clouddriver-oracle-bmcs.gradle

```diff
@@ -0,0 +1,52 @@
+// Oracle BMCS SDK isn't published to any maven repo (yet!), so we manually download, unpack and add to compile/runtime deps
+def bmcsSDK = project.file('bmcs-sdk/lib/oracle-bmc-java-sdk-full-1.2.2.jar')
```

Comment:
Good catch! Added a cleanSDK task and added it as a dependency for the clean task.